sample/src/main/java/com/polidea/rxandroidble2/sample/example2_connection/ConnectionExampleActivity.java

```diff
@@ -58,15 +54,12 @@ public void onConnectToggleClick() {
     @TargetApi(Build.VERSION_CODES.LOLLIPOP)
     @OnClick(R.id.set_mtu)
     public void onSetMtu() {
-        final Disposable disposable = bleDevice.establishConnection(false)
+        mtuDisposable = bleDevice.establishConnection(false)
```

Comment:
It is possible that the `mtuDisposable` will get overwritten if user would tap the button more than once